cmd/kubelet/app/server.go

```diff
@@ -674,7 +674,7 @@ func RunKubelet(kcfg *KubeletConfig) error {
 		return fmt.Errorf("failed to create kubelet: %v", err)
 	}
 
-	util.ApplyRLimitForSelf(kcfg.MaxOpenFiles)
+	resourcecontainer.ApplyRLimitForSelf(kcfg.MaxOpenFiles)
```

Comment:
This function is very poorly named, and it is used in exactly one place - kubelet.  Can we just move it to kubelet and call it rlimitNumFiles()